### PR TITLE
Add new types

### PR DESCRIPTION
**Pull request by ydlamba**
Required in the editor for data inspector feature.

### CODE REVIEW

File: types/runtime/index.d.ts [hunk 1]
```diff
@@ -54,6 +56,9 @@ export class View {
   addResizeListener(handler: ResizeHandler): View;
   removeResizeListener(handler: ResizeHandler): View;
   tooltip(handler: TooltipHandler): View;
+
+  getState(options?: any): any;
```
Comment (domoritz) [inline types/runtime/index.d.ts]: Can you make the options more specific?

Comment (ydlamba) [reply types/runtime/index.d.ts]: You mean `object`? or even more specific?

Comment (domoritz) [reply types/runtime/index.d.ts]: More specific

Comment (ydlamba) [reply types/runtime/index.d.ts]: Done.

File: types/runtime/index.d.ts [hunk 2]
```diff
@@ -54,6 +56,13 @@ export class View {
   addResizeListener(handler: ResizeHandler): View;
   removeResizeListener(handler: ResizeHandler): View;
   tooltip(handler: TooltipHandler): View;
+
+  getState(options?: {
+    signal?: (name?: string, operator?: any) => boolean;
+    data?: (name?: string, object?: any) => boolean;
+    recurse?: boolean;
+  }): { signal: any; data: any };
```
Comment (domoritz) [inline types/runtime/index.d.ts]: `signal?: any; data?: any`

Comment (ydlamba) [reply types/runtime/index.d.ts]: Oops.

**Pull request merged**